napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -380,6 +380,10 @@ def add_vertex_to_path(
     vertices = layer._data_view.shapes[index].data
     vertices = np.concatenate((vertices, [coordinates]), axis=0)
     value = layer.get_value(event.position, world=True)
+    # If there was no move event between two clicks value[1] is None
+    # and needs to be taken care of.
+    if value[1] is None:
+        value = layer._moving_value
```

Comment:
Does any of this have a negative effect on the `layer._last_cursor_position = np.array(event.pos)` line in the `add_vertex_to_path` function? That's the only line in this function that didn't originally exist when this PR was first created. I assume the tests will pick up any problems?

Reply:
Yes, it won't be reset to `None` after completing drawing the path when clicking at the same position twice with `Add path` enabled.